gui/symfony-docker/src/Controller/EquarrisseurController.php

```diff
@@ -96,16 +99,30 @@ public function list(ResourcesListHandler $listHandler,
         // }
 
         return $this->render('pro/equarrisseur/list.html.twig',
-            ['resources' => $resources]
+            ['resources' => $resources,
+                'category' => $category]
         );
     }
 
-    #[Route('/specific/{id}', name: 'app_equarrisseur_job')]
-    public function job($id): Response
+    #[Route('/specific/{id}/{category}', name: 'app_equarrisseur_job')]
+    public function job($id,$category): Response
     {
+        switch (strtolower($category)) {
+            case 'animal':
+                $category = "Carcass";
+                break;
+            case 'carcasse':
+                $category = "Demi Carcass";
+                break;
+            case 'Demi%20Carcasse':
+                $category = "Demi Carcasse";
```

Comment:
this case is the same that the previous one, it is normal ? 
If yes, please set a default case with the $category ="Demi carcasse" to have good practice